### PR TITLE
feat: normalize

### PR DESCRIPTION
**Pull request by mjmonline**
The small amount of actual elements targeted by our trimmed normalize can live in a global css in my opinion

### CODE REVIEW

**Comment by mjmonline** [comment]
@iamstarkov Looking at the CSS in this PR it's still bloated, we really only need to normalize `<button>`, `<input>`, `<select>` and `<textarea>`. And the normalize of these elements should live inside the core components themselves

**Comment by iamstarkov** [comment]
hmm 🤔, now its just a button styles

File: src/Molecules/Button/Button.tsx [hunk 1]
```diff
@@ -50,7 +50,39 @@ const getHeight = (props: ThemedStyledProps<Props, Theme>) => {
...
+  line-height: 1.15;
+  margin: 0;
+  overflow: visible;
+  text-transform: none;
+
+  &,
+  &[type='button'],
+  &[type='reset'],
+  &[type='submit'] {
+    -webkit-appearance: button; /* stylelint-disable-line */
```
Comment (JacobBlomgren) [inline src/Molecules/Button/Button.tsx]: just a noob question, what does styled components do with already prefixed stuff, like this? might it be stripped out?

File: src/Molecules/Button/Button.tsx [hunk 2]
```diff
@@ -50,7 +50,39 @@ const getHeight = (props: ThemedStyledProps<Props, Theme>) => {
   return `${theme.spacing.unit(hugeness)}px`;
 };
 
-const StyledButton = styled.button<Props>`
+/** From Normalize.css v8.0.1 */
+const NormalizedButton = styled.button`
```
Comment (JacobBlomgren) [inline src/Molecules/Button/Button.tsx]: Could this be moved to utils? I have `Tab` component right now in `Tabs`, that probably can't and shouldn't reuse `Button` (though up for discussion), and I would like to reuse this then.

File: src/Molecules/Button/Button.tsx (hunk 1 above)
Comment (mjmonline) [reply src/Molecules/Button/Button.tsx]: This should be ignored and printed out as is

Comment (mjmonline) [reply src/Molecules/Button/Button.tsx]: > hmm 🤔, now its just a button styles

Using individual resets within every component seems more in harmony with the styled-components philosophy to me

File: src/Molecules/Button/Button.tsx (hunk 2 above)
Comment (mjmonline) [reply src/Molecules/Button/Button.tsx]: Moved to comon folder

**Comment by iamstarkov** [comment]
👍

**Pull request merged**